src/Client.php

```diff
@@ -4,12 +4,12 @@
 
 use Exception;
 use MongoDB\BSON;
-use Swoole\Client;
+use Swoole\Client as SwooleClient;
```

Comment:
What benefits do we get from the Swoole TCP client? Can we have an adapter pattern for this?